package.json

```diff
@@ -23,9 +23,9 @@
   "build": {
     "appId": "gov.nrel.neb-tool",
```

Comment:
Should we change to gov.nrel.justifi?

Reply:
I don't think this is actively used until we are doing releases and then it will most likely change but we can change it for now